polaris-react/src/utilities/get-aria-attributes.ts

```diff
@@ -0,0 +1,7 @@
+import type React from 'react';
+
+export function getAriaAttributes<T extends React.AriaAttributes>(props: T) {
+  return Object.fromEntries(
+    Object.entries(props).filter(([prop]) => prop.startsWith('aria-')),
+  );
```

Comment:
Was thinking about this as well. Why would the consumer even use Box in that case and not just a div?

Reply:
We are talking about allowing this across all our layout components though, so that could be used on `Columns` or `Card` and then things can start getting unpredictable.

Reply:
If a team needs to do this then it means there's a gap in our system and we might want to address it. 

There will always be people breaking out of the system but I'm not concerned about them using this ts ignore 'weakness' as the way they'd do that.

We'll be thinking more about escape hatches in the new year and we should think about these things at that time, not now.